index.html

```diff
@@ -1,361 +1,438 @@
+              <li><a href="">About</a></li>
+              <li><a href="">Contact</a></li>
+              <li>
+                <a href=""><img src="img/envelop.svg" alt="Email" /></a>
+              </li>
+            </ul>
+          </nav>
         </div>
       </header>
-      
-    <h1 class="primary-text">
-        Hey There. <br>I’m Yaseen
-    </h1>
-    <h2 class="supporting-text">
-        I’am a Software Developer
-    </h2>
-    <p class="discription">
-        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate 
-        <br class="br">to contact me.
-    </p>
-    <ul class="social-media">
-        <li  class="social-media-buttons">
-            <a href="https://github.com/">
-                <img class="social-media-icons cat-icon" src="img/cat.svg" alt="">
-            </a>
+
+      <h1 class="primary-text">Hey There. <br />I’m Yaseen</h1>
+      <h2 class="supporting-text">I’am a Software Developer</h2>
+      <p class="discription">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate
+        <br class="br" />to contact me.
+      </p>
+      <ul class="social-media">
+        <li class="social-media-buttons">
+          <a href="https://github.com/">
+            <img
+              class="social-media-icons cat-icon"
+              src="img/cat.svg"
+              alt="GitHub"
+            />
+          </a>
         </li>
-        <li  class="social-media-buttons">
-            <a href="https://www.linkedin.com/">
-                <img class="social-media-icons" src="img/in.png" alt="">
-            </a>
-            </li>
-        <li  class="social-media-buttons">
-            <a href="">
-                <img class="social-media-icons" src="img/hand.png" alt="">
-            </a>
+        <li class="social-media-buttons">
+          <a href="https://www.linkedin.com/">
+            <img class="social-media-icons" src="img/in.png" alt="linkedin" />
+          </a>
         </li>
-        <li  class="social-media-buttons">
-            <a href="https://twitter.com">
-                <img class="social-media-icons" src="img/bird.png" alt="">
-            </a>
+        <li class="social-media-buttons">
+          <a href="">
+            <img
+              class="social-media-icons"
+              src="img/hand.png"
+              alt="Angel-list"
+            />
+          </a>
         </li>
-        <li  class="social-media-buttons">
-            <a href="">
-                <img class="social-media-icons" src="img/m.png" alt="">
-            </a>
+        <li class="social-media-buttons">
+          <a href="https://twitter.com">
+            <img class="social-media-icons" src="img/bird.png" alt="Twitter" />
+          </a>
         </li>
-    </ul>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you also fix the below **overlapping text** in the home section on some screen size to make your page looks more professional and still stick to the Figma design
![Screenshot from 2022-07-19 17-12-08](https://user-images.githubusercontent.com/83042742/179786352-1654a1fc-8ff5-4161-9d23-466f81334a6f.png)
.